content/posts/hiero_links_of_april_twentyseven.md

```
@@ -0,0 +1,29 @@
+
+---
+
+**ivaylonikolov7** implemented High Volume Entity Creation [HIP 1313](https://github.com/hiero-ledger/hiero-sdk-js/pull/3974) in the JavaScript SDK to enable efficient batch creation of large numbers of entities. 
+
+**Keshav Kumar Sharma** added a [good first issue guidelines](https://github.com/hiero-ledger/hiero-sdk-java/pull/2723) for the Java SDK to define what is and is not appropriate for beginner contributors. 
+
+**Dosi Kolev** created an updated [release](https://github.com/hiero-ledger/hiero-sdk-go/pull/1689) of the Go SDK to document new changes and updates. 
+
+**Matt Hess** updated [test constructor](https://github.com/hiero-ledger/hiero-consensus-node/pull/25198) invocation to correct a signature mismatch compilation error in Hiero Consensus Node. 
+
+**Want to be featured on the Hiero-Website?**
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)